CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Git LFS Changelog
 
+## 2.0.1 (6 March, 2017)
+
+### Misc
+
+* tq: fallback to \_links if present #2007 (@ttaylorr)
```

Comment:
In the future, I'd just wrap backticks around it like:

```
`_links`
```

Escaping with a `\` is technically correct markdown, but backticks are more readable in plain markdown, and in rendered html.